docs/docs-ref-autogen/excel/excelscript/excelscript.datavalidation.yml

```
@@ -209,4 +209,42 @@ methods:
           type: '<xref uid="ExcelScript!ExcelScript.DataValidationRule:interface" />'
       return:
         type: void
-        description: ''
+        description: |-
+
+
+          #### Examples
+
+          ```TypeScript
+          /**
+           * This script creates a data validation rule for the range B1:B5.
+           * All values in that range must be a positive number.
+           * Attempts to enter other values will be blocked and an error message will appear.
```

Should this be present tense?
```suggestion
           * Attempts to enter other values are blocked and an error message appears.
```